docs/integrations/ai-engines/monkeylearn.mdx

```diff
@@ -54,7 +59,9 @@ SELECT * FROM mindsdb.models WHERE name = 'ecommerce_ticket_classifier';
 
```

Comment:
Let's use `DESCRIBE ecommerce_ticket_classifier` instead of selecting from models.